sqlglot/dialects/postgres.py

```diff
@@ -32,6 +32,22 @@ def func(self, expression):
     return func
 
 
+def _distance_sql(self, expression):
+    return self.binary(expression, "<->")
+
+
+def _lateral_sql(self, expression):
+    this = self.sql(expression, "this")
+    if isinstance(expression.this, exp.Subquery):
+        return f"LATERAL{self.sep()}{this}"
+    alias = expression.args["alias"]
+    table = alias.name
```

Comment:
can you do expression.text("alias") ?

Reply:
Doing this means that alias would become a string and then `alias.name` etc wouldn't work, right?

Reply:
yea but why do you need alias.name j
if you have it from the text?

Reply:
Besides that there's also the self.expressions call which uses the alias expression object too.

Reply:
yea just do that

Reply:
Also, is expression.text("alias") correct? I think this always results in an empty string here.

Reply:
not sure. do what works i guess